src/httputil.js

```diff
@@ -0,0 +1,37 @@
+class HttpUtil {
+  constructor() {}
+
+  getHttpRequest() {
+    if (window.XMLHttpRequest) {return new XMLHttpRequest();}
```

Comment:
Esto no se iba a eliminar?

Reply:
You right, i dont know really why it is there again... propably becouse the checkout use it in the original implement but i have removed this dependence so I will remove it again.